courses/level102/containerization_and_orchestration/orchestration_with_kubernetes.md

```diff
@@ -76,7 +76,7 @@ This workflow might help you understand the working on components better:
 
 ### Prerequisites
 
-The best way to start this exercise is to use a [Katacoda kubernetes playground](https://www.katacoda.com/courses/kubernetes/playground). A single node kubernetes cluster is already set up for you here for quick experimentation. You can also use this to play with docker.
+The best way to start this exercise is to use a [Play with kubernetes lab](https://labs.play-with-k8s.com/). 
 
 The environment gets torn down after 10 mins. So make sure that you save your files if you want to resume them. For persistent kubernetes clusters, you can set it up either in your local (using [minikube](https://minikube.sigs.k8s.io/docs/start/)) or you can create a [kubernetes cluster in Azure](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal), GCP or any other cloud provider.
```

Comment:
Is this (10 min tear down) relevant for Play with Kubernetes lab too?

Reply:
Teardown for play with k8s lab is 4 hours, overlooked it by mistake, have updated it in another commit